Week3/homework/1-step3.js

```diff
@@ -2,7 +2,10 @@
 
 function foo(func) {
   // What to do here? 
+  //func();
+  func = func + 1
 }
+func();
```

Comment:
Hi @Gaffari02 - I am not sure what are you trying to do here. Let's go through the exercise again. It says `We saw that we can pass functions as arguments to other functions. Your task is to write a function that takes another function as an argument and runs it.`. Which means that `foo` should be expecting a function, possibly `bar` and execute it. `bar` here would be our callback.